src/Oro/Bundle/UserBundle/Controller/UserController.php

```diff
@@ -87,9 +88,9 @@ public function createAction()
      * @Template
      * @AclAncestor("pim_user_user_edit")
      */
-    public function updateAction(User $entity)
+    public function updateAction($id)
```

Comment:
Why you have to do this ? in the update method you fetch the User so you could let the ParamConverter do his job, am I wrong ?